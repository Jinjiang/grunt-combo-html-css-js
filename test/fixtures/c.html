<!DOCTYPE html>
<html>
<head>

<title>Test</title>
<link rel="stylesheet" type="text/css" href="style.css">
<link rel="stylesheet" type="text/css" ignore="true" href="style.css">

</head>
<body>

<h1>An Empty Page with <strong>internal</strong> css &amp; javascript files.</h1>
<script type="text/javascript" src="script.js"></script>
<script type="text/javascript" src="script.js" ignore></script>
<script type="text/javascript" src="script.js" ignore="false"></script>

</body>
</html>
