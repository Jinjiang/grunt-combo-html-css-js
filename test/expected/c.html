<!DOCTYPE html>
<html>
<head>

<title>Test</title>
<style>
body {background-color: #f0f0f0;}
</style>
<link rel="stylesheet" type="text/css" ignore="true" href="style.css">

</head>
<body>

<h1>An Empty Page with <strong>internal</strong> css &amp; javascript files.</h1>
<script>
document.write('hello world');
</script>
<script type="text/javascript" src="script.js" ignore></script>
<script>
document.write('hello world');
</script>

</body>
</html>
